Core/include/Acts/TrackFitting/KalmanFitter.hpp

```diff
@@ -152,6 +153,10 @@ struct KalmanFitterOptions {
   /// ReverseFilteringLogic
   bool reversedFiltering = false;
 
+  /// Factor by which the covariance of the input of the reversed filtering is
+  /// scaled
+  double reversedFilteringCovarianceScaling = 1.0;
```

Comment:
Can we make this absolutely clear that it's only used in case we actually do the backward propagation.